modules/internal/DefaultSparse.chpl

```diff
@@ -239,7 +239,7 @@ module DefaultSparse {
     proc bulkAdd_help(inds: [?indsDom] index(rank, idxType), dataSorted=false,
         isUnique=false){
 
-      bulkAdd_prepareInds(inds, dataSorted, isUnique);
+      bulkAdd_prepareInds(inds, dataSorted, isUnique, Sort.defaultComparator);
```

Comment:
We've clearly had it for a very long time, but today I'm finding myself surprised that we defined '>' and '<' on tuples given that one might have different definitions of what less than means in a row-major vs. column-major order (as one example).  Just curious if you found that choice of a default surprising as well...  (as compared to not providing a default at all).